packages/types/index.ts

```diff
@@ -23,7 +23,7 @@ import type { ListPaymentsParams } from './modules/list_payments:params';
 import type { ListPaymentsResponse } from './modules/list_payments:response';
 import type { CreateTransactionParams } from './modules/create_transaction:params';
 import type { CreateTransactionResponse } from './modules/create_transaction:response';
-import type { CheckoutBody } from './modules/@checkout:params';
+import type { CheckoutBody, Transaction } from './modules/@checkout:params';
```

Comment:
Isso seria interno na verdade..

Reply:
Acho que eu usaria `CheckoutBody['transaction']` porque não tenho certeza se o `json-schema-to-typescript` vai continuar expondo a interface `Transaction`, a prática ele gera ela pra usar internamente

Reply:
> Mas não chega a ser um problema aqui não